roles/libvirt_manager/tasks/start_manage_vms.yml

```diff
@@ -54,7 +54,7 @@
       Host {{ vm_ip.nic.host }} cifmw-{{ vm_ip.nic.host }} {{ extracted_ip }}
         ProxyJump {{ ansible_user | default(lookup('env', 'USER')) }}@{{ proxy_hostname }}
         Hostname {{ extracted_ip }}
-        User {{ 'core' if vm_ip.nic.host.startswith('crc') else 'zuul' }}
+        User {{ 'core' if vm_ip.nic.host is match('^(crc|ocp).*') else 'zuul' }}
```

Comment:
so this and the other code like it is making sure we use the core user for vms deploy by the ci framework if ht hostname starts with crc or ocp.

would it not be better to just have a way to map the user to hostname?

by the way zuul jobs are not meant to assume they are logging in as the zuul user either for portability reasons. but that fine for now i guess.

the main thing to ensure is that we do not just use root when sshing in.
so I'm happy that we are using core or zuul and not root.

i would have just tried to make this contolabel via the inventory file using host/group vars instead but this is functionl

Reply:
so there's a way to pass the `admin_user` from the configuration dict - but there are some other issues with that. I had to hack my way in the past, maybe a follow-up would help converging again with that parameter (not really the scope of this PR)